indico/modules/events/reminders/models/reminders.py

```diff
@@ -255,7 +261,7 @@ def all_recipients(self):
         This includes both explicit recipients and, if enabled,
         participants/speakers of the event.
         """
-        recipients = set(self.recipients)
+        recipients = {Recipient(None, None, email) for email in self.recipients}
```

Comment:
with a dataclass that has defaults for the names and the email as the first field you can avoid this ugly `None, None` :)